deploy/lib/server_config.rb

```diff
@@ -2058,6 +2058,37 @@ def deploy_rest
       deploy_rest_config()
       deploy_ext()
       deploy_transform()
+
+      if !test &&
```

Comment:
No, deploy_rest calls itself to repeat the deploy stuff against test-port. See following lines, I create a new temporary mlRest client, targeting test port and such, and then make a recursive call with a true flag to stop further recursion..